Tella/Scenes/Settings/Views/Servers/Uwazi/Step2/UwaziServerAccessSelectionView.swift

```diff
@@ -0,0 +1,100 @@
+//
+//  UwaziServerAccessSelectionView.swift
+//  Tella
+//
+//  Created by Robert Shrestha on 4/18/23.
+//  Copyright © 2023 INTERNEWS. All rights reserved.
+//
+
+import SwiftUI
+
+struct UwaziServerAccessSelectionView: View {
+    @State var isButtonValid = true
+    @Environment(\.presentationMode) var presentationMode: Binding<PresentationMode>
+    // TODO: Ask if there is any good way to do it
+    @State var isLoginSelected: Bool = false
+    @State var isPublicInstance: Bool = false
+
+    @EnvironmentObject var serversViewModel : ServersViewModel
+    @EnvironmentObject var serverViewModel : UwaziServerViewModel
+    @EnvironmentObject var mainAppModel : MainAppModel
+    var body: some View {
+        ContainerView {
+            VStack {
+                Spacer()
+                Image("settings.server")
+                VStack(spacing: 12) {
+                    Text(LocalizableSettings.UwaziAccessServerTitle.localized)
+                        .font(.custom(Styles.Fonts.regularFontName, size: 18))
+                        .foregroundColor(.white)
+                        .multilineTextAlignment(.center)
+                    Spacer().frame(height:24)
+                    TellaButtonView<AnyView>(title: LocalizableSettings.UwaziLogin.localized,
+                                             nextButtonAction: .action,
+                                             isValid: .constant(true),action: {
+                        self.isLoginSelected = true
+                        self.isPublicInstance = false
+                    })
+                    .overlay( self.isLoginSelected ?
+                              RoundedRectangle(cornerRadius: 20)
+                        .stroke(.white, lineWidth: 4) : nil
+                    )
+                    TellaButtonView<AnyView>(title: LocalizableSettings.UwaziPublicInstance.localized,
+                                             nextButtonAction: .action,
+                                             isValid: .constant(true),action: {
+                        self.isLoginSelected = false
+                        self.isPublicInstance = true
+                    })
+                    .overlay( self.isPublicInstance ?
+                              RoundedRectangle(cornerRadius: 20)
+                        .stroke(.white, lineWidth: 4) : nil
+                    )
+                }.padding(EdgeInsets(top: 0, leading: 24, bottom: 0, trailing: 24))
+                Spacer()
+                BottomLockView<AnyView>(isValid: $isButtonValid,
+                                        nextButtonAction: .action,
+                                        nextAction: {
+                    if isLoginSelected {
+                        let loginView = UwaziLoginView()
```

Comment:
@rob-horizontal-iOS Could you put this code into a separate function called for example `showUwaziLoginView`